### DISCUSSION TITLE
Starship thinks my home directory is a python project?

### DISCUSSION THREAD

**Discussion by ashleysommer** [starship/starship#1620] [Q&A] [answered]
First time using Starship today.
Upon launching starship in my home directory (using ZSH), I see
```
~ via 🐍 v2.7.18rc1
>
```
Seems the inclusion of the Snake there indicates that starship thinks my home directory is a Python project? There are certainly quite a few python projects in my account, but they are all in `~/CODE/*` or `~/PythonProjects/*`.

I've searched through all of the files in my home directory to try to determine why the prompt would think my home directory is a python project.

I'm after some advice about what kind of stray files I might have laying around in my home directory that might cause starship to do this.
I've checked for `setup.py`, `requirements.txt`, `pyproject.toml`, `setup.cfg`, `venv/`, `.venv/`, `dist/`, `build/`, `MANIFEST.in`, even for `README.rst`, but none of these exist at the root level of my home directory.

**Comment by andytom** [comment, marked as answer]
Hi @ashleysommer, the python module is shown when the directory contains any of the following files `.python-version`, `requirements.txt`, `pyproject.toml`, `Pipfile`, `tox.ini`, `setup.py `, `__init__.py` or a file with the `.py` extension. Do you see any of those files?

**Reply by ashleysommer** [reply]
Hi @andytom 
Thanks. Yes, that helped. I found a stray `*.py` file in my home directory.
Removing that file did remove the snake icon from the prompt line.
I still think that is a strange choice to mark any directory with a `*.py` as a Python module. Lots of OS directories have `*.py` files for scripting purposes, and I don't think its logical to call that directory a Python module.

**Reply by andytom** [reply]
Just to clarify when I say module I mean the starship module not a python module. If you want you can use the `scan_for_pyfiles` option to stop the python module from activating when the folder contains a file with the `.py` extension see https://starship.rs/config/#python.

**Reply by ashleysommer** [reply]
Oh, I didn't see that option in the config docs. Thanks, that's exactly what I was after. Setting scan_for_pyfiles to false fixes my complaint.
